src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -50,6 +50,16 @@
 }
 
 .unified-share-modal {
+    /** title and classifiction label */
+    .bdl-UnifiedShareModalTitle {
+        display: flex;
+        flex-flow: row wrap;
+
+        .bdl-UnifiedShareModalTitle-classification {
```

Comment:
With suit you dont have to nest that much btw, you can have this one one level above like

.bdl-UnifiedShareModalTitle {
    display: flex;
    flex-flow: row wrap;
}

.bdl-UnifiedShareModalTitle-classification {
    margin-left: 6px;
}